Cargo.toml

```diff
@@ -11,38 +11,29 @@ version = "0.37.0"
 edition = "2018"
 
 [dependencies]
-async-recursion = "0.3.1"
 bincode = "1.2.1"
 bls = { package = "threshold_crypto", version = "~0.4.0" }
 bls_dkg = "~0.2.0"
 bls_signature_aggregator = "~0.1.4"
 bytes = "~0.5.4"
-crossbeam-channel = { version = "~0.4.2", optional = true }
-ctrlc = { version = "3", optional = true, features = ["termination"] }
 ed25519-dalek = { version = "1.0.0-pre.3", features = ["serde"] }
-env_logger = { version = "~0.7.1", optional = true }
 err-derive = "~0.2.4"
-sn_fake_clock = "~0.4.0"
-futures = "~0.3.5"
+futures = "~0.3.6"
 hex_fmt = "~0.3.0"
 itertools = "~0.9.0"
-lazy_static = { version = "1", optional = true }
 log = "~0.4.8"
 lru_time_cache = "~0.11.0"
-mock-quic-p2p = { git = "https://github.com/maidsafe/qp2p", rev = "602dc620", optional = true }
-qp2p = { git = "https://github.com/maidsafe/qp2p", features = ["upnp"], branch = "master" }
+qp2p = { version = "~0.8.4", features = ["upnp"] }
 rand = "~0.7.3"
-rand_core = "~0.5.1"
-rand_os = "~0.2.2"
 rand_xorshift = "~0.2.0"
-serde = { version = "1.0.111", features = ["derive" ,"rc"] }
```

Comment:
seems there is compile error for testing code?
```
error[E0463]: can't find crate for `serde_derive` which `serde` depends on
  --> src/lib.rs:70:1
   |
70 | extern crate serde;
   | ^^^^^^^^^^^^^^^^^^^ can't find crate
```

Reply:
No idea what's up with this error. Seems to happen only on macos.

Reply:
Seems to be fixed now.